lib/extend.js

```diff
@@ -75,7 +76,7 @@ var extend = function extend() {
 /**
  * @public
  */
-extend.version = '1.1.3';
```

Comment:
I think rather than doing that, it'd be better to just drop the export entirely. Runtime code doesn't need to know what the version is.